src/web.cpp

```diff
@@ -436,12 +443,11 @@ void handle_status()
     ADD_INT(json, "TTCseconds", TTCdelay);
     END_JSON(json);
 
-    RINFO("Status requested:\n%s", json);
+    RINFO("Status requested:\n%s\nlength %d\n", json, strlen(json));
```

Comment:
You don't need to keep this change, I was just curious how close we were to filling up that string

Reply:
True, but no harm to leave it.  Is useful I think.